dq-game/src/client/event_coordinator.py

```diff
@@ -21,12 +22,16 @@ def __init__(self, controller, renderer):
     def on_timeout(self):
         self.renderer.screen_handler.clear_data()
         self.renderer.show_logo()
+        if self.front_screen:
```

Comment:
When it will happen that the event coordinator for the client has no front_screen

Reply:
you never know...